draft-ietf-quic-manageability.md

```diff
@@ -531,7 +522,7 @@ see also {{?RFC7605}} section 5.
 
 While the second most significant bit (0x40) of the first octet is set to
 1 in most QUIC packets of the current version (see {{public-header}}),
-this method of recognizing QUIC traffic is NOT RECOMMENDED. First, it only
+this method of recognizing QUIC traffic is not recommended. First, it only
```

Comment:
```suggestion
this method of recognizing QUIC traffic is NOT RECOMMENDED. First, it only
```

Proposed another fix in PR #223. So reverting to avoid merge conflict....